frontend/src/scenes/sceneLogic.ts

```diff
@@ -441,8 +441,12 @@ export const sceneLogic = kea<sceneLogicType>([
             }
         }
         for (const [path, scene] of Object.entries(routes)) {
-            mapping[path] = (params, searchParams, hashParams, { method }) =>
-                actions.openScene(scene, { params, searchParams, hashParams }, method)
+            mapping[path] = (params, searchParams, hashParams, { method }) => {
+                if (path === urls.dataWarehouse() && featureFlagLogic.values.featureFlags[FEATURE_FLAGS.SQL_EDITOR]) {
```

Comment:
I think having a separate URL for the SQL editor is fine, don't think it makes much sense to have the new SQL editor under `/data-warehouse`?